### ISSUE TITLE
Improve IDE boot performance

### ISSUE THREAD

**Issue by teozkr** [PicoNodes/PicoNodes#30]
The biggest gain here would probably be updating to an Akka.JS version where akka-js/akka.js#82 is fixed (probably 1.2.5.13?).

**Comment by teozkr** [comment]
Postponed this for now, since we're waiting for upstream.

**Comment by andreaTP** [comment]
yes I'll merge and I will release it with next `1.2.5.13` Akka team is usually quite fast in releasing.

**Comment by teozkr** [comment]
Great to know. Once again, thanks!
